__tests__/internal/unit/server-test.js

```diff
@@ -73,6 +73,14 @@ describe("Unit | createServer", function () {
 
     server.shutdown();
   });
+
+  test("forces timing to be 0 in test environment", () => {
+    let server = new Server({ environment: "test" });
+
+    expect(server.timing).toEqual(0);
```

Comment:
Just to confirm, this test fails without your changes in this PR?

Reply:
With the proposed breaking change, this should expect undefined.

Reply:
Correct :+1: